randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -1262,11 +1274,25 @@ Extra - room_id: [36]
   * Extra - door_idx: (7,)
   > Door to Tourian Shaft East
       All of the following:
-          Morph Ball and Screw Attack and Can Bounce in Ball
+          Morph Ball and Screw Attack
           Any of the following:
-              # Get past Rippers - TODO: health for damage runs
-              Space Jump or Damage Runs (Intermediate)
+              # Get past Rippers
+              Space Jump
+              All of the following:
+                  Damage Runs (Beginner)
+                  Any of the following:
+                      All of the following:
```

Comment:
🔧 Should probably have a comment that this is for just jumping in and out of the acid